lib/screens/settings/follow_event_config.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/messages/twitch/eventsub_configuration.dart';
+
+class FollowEventConfigWidget extends StatelessWidget {
+  const FollowEventConfigWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text("Follow Config"),
+        ),
+        body: Consumer<EventSubConfigurationModel>(
+            builder: (context, model, child) {
+          return Column(
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(16),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text("Pin Duration",
+                        style: TextStyle(
+                          color: Theme.of(context).accentColor,
+                          fontWeight: FontWeight.bold,
+                        )),
+                    Slider.adaptive(
+                      value: model.followEventConfig.eventDuration,
+                      min: 2,
+                      max: 14,
+                      divisions: 4,
+                      label: "${model.followEventConfig.eventDuration}seconds",
+                      onChanged: (value) {
+                        model.setFollowEventDuration(value);
+                      },
+                    ),
+                    SwitchListTile.adaptive(
+                      title: const Text('show event'),
```

Comment:
Should we follow the capitalization 'standard' that has been used in the settings? "Enable event"/"Show event"

Reply:
Agree.